src/app/games/lasca/LascaTutorial.ts

```diff
@@ -0,0 +1,78 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { Coord } from 'src/app/jscaip/Coord';
+import { LascaMove } from './LascaMove';
+import { LascaPiece, LascaSpace, LascaState } from './LascaState';
+
+const zero: LascaPiece = LascaPiece.ZERO;
+const one: LascaPiece = LascaPiece.ONE;
+const _u: LascaSpace = new LascaSpace([zero]);
+const _v: LascaSpace = new LascaSpace([one]);
+const __: LascaSpace = LascaSpace.EMPTY;
+
+export class LascaTutorial extends Tutorial {
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            $localize`Lasca: origins`,
+            $localize`Lasca is a game based on draughts created in 1911 by Emanuel Lasker, chess world champion. It's played on a 7x7 board, each player has 11 pieces.`,
+            LascaState.getInitialState(),
+        ),
+        TutorialStep.informational(
+            $localize`Goal of the game`,
+            $localize`The goal of Lasca is, like for draughts, to render the opponent out of possible move, either by capturing all his pieces, either by blocking them.`,
+            LascaState.getInitialState(),
+        ),
+        // 3. Simple step
+        TutorialStep.anyMove(
+            $localize`Steps`,
+            $localize`A simple step is made by one diagonal move forward, left or right. Click on the chosen piece, then on it's landing square.<br/><br/>You are playing Dark, do the first move.`,
+            LascaState.getInitialState(),
+            LascaMove.fromStep(new Coord(4, 4), new Coord(3, 3)).get(),
+            $localize`Congratulations!`,
+        ),
+        // 4. Simple capture
+        TutorialStep.anyMove(
+            $localize`Capture`,
+            $localize`You have to capture when you can. It is the case here, so click on the piece that must capture, and then on its landing square.<br/><br/>You're playing Dark, go ahead.`,
```

Comment:
J'suis pas fan du « yu have to capture when you can », ça peut être compris par « tu peux capturer quand c'est légal ». Peut-être inverser: as soon as you can capture, you must capture

Reply:
"ça peut être compris par 'tu peux capturer quand c'est légal' "" bah c'est très exact, comme interpretation ?
le "as soon" ça fait spécial, c'est du tout par tour.

Une autre suggestion ?